ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldLRspec.json

```diff
@@ -4983,5 +4983,43 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "respondent1Link",
```

Comment:
Can this be renamed to something like respondent1DocumentURL?

Reply:
Done